pkg/ansible/controller/status/types.go

```diff
@@ -165,3 +166,18 @@ func CreateFromMap(statusMap map[string]interface{}) Status {
 	}
 	return Status{Conditions: conditions}
 }
+
+// GetJSONMap - gets the map value for the status object.
+// This is used to set the status on the CR.
+// This is needed because the unstructured type has special rules around DeepCopy.
+// If you do not convert the status to the map, then DeepCopy for the
+// unstructured will fail and throw runtime exceptions.
+func (status *Status) GetJSONMap() map[string]interface{} {
+	m := map[string]interface{}{}
+	b, err := json.Marshal(status)
+	if err != nil {
+		return m
```

Comment:
Is it ok to ignore the error here and just return an empty map?

Reply:
as discussed offline adding a log and mentioning that it eats the error on the doc.